DeepSpeech.py

```diff
@@ -724,6 +724,17 @@ def do_graph_freeze(output_file=None, output_node_names=None, variables_blacklis
         if not FLAGS.export_tflite:
             frozen_graph = do_graph_freeze(output_node_names=output_names, variables_blacklist='previous_state_c,previous_state_h')
             frozen_graph.version = int(file_relative_read('GRAPH_VERSION').strip())
+
+            # Add a no-op node to the graph with metadata information to be loaded by the native client
```

Comment:
And we don't break with TF Lite? nice :)

Reply:
Nope. We don't get the info in the TFLite graph either, unfortunately. I'll try to find a workaround when I have some time.